images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -8,29 +9,42 @@ ARG AGENTBUILD
 # NOTE: This imge is NOT based on 'githubrunners-buildpackdeps-stretch' because AzFunctions
 # team wanted a consistent experience at their end and do not want to see latency that might
 # be caused due to when the GitHub runners' layers and Oryx image layers go out of sync.
-FROM githubrunners-buildpackdeps-stretch AS main
+FROM githubrunners-buildpackdeps-${DEBIAN_FLAVOR} AS main
 ARG BUILD_DIR
 ARG IMAGES_DIR
+ARG DEBIAN_FLAVOR
 
 # Configure locale (required for Python)
 # NOTE: Do NOT move it from here as it could have global implications
 ENV LANG C.UTF-8
 
 # Install basic build tools
 RUN apt-get update \
-    && apt-get upgrade -y \
```

Comment:
why remove this? It will help us get security patches, right? Or are you counting on the base image always having latest bits?